grub-btrfsd

```diff
@@ -206,7 +206,7 @@ setup() {
 create_grub_menu() {
     #  create the grub submenu of the whole grub menu, depending on wether the submenu already exists
     #  and gives feedback if it worked
-    if grep "snapshots-btrfs" "{grub_directory}/grub.cfg"; then
+    if grep "snapshots-btrfs" "${GRUB_BTRFS_GRUB_DIRNAME:-/boot/grub}"/grub.cfg; then
```

Comment:
I am not sure how this solves it. The `grub_directory` variable should already be set to this:
https://github.com/Antynea/grub-btrfs/blob/ade5a91d5554cca3c0cbb48e2b16d0db41887a8a/41_snapshots-btrfs#L93